libs/blocks/quiz-results/quiz-results.js

```diff
@@ -13,6 +14,7 @@ async function loadFragments(el, experiences) {
     el.append(a);
     await createFragment(a);
   }
+  document.querySelectorAll('main > div').forEach((section, idx) => decorateSectionAnalytics(section, idx, getConfig()));
```

Comment:
we've removed the duplicate section numbers, but not the duplicate block number.  You have a unique situation that is normally not allowed: a block nested in a block. I think it will fix it if you add this code after line 17:
`document.querySelectorAll('.quiz-results').forEach((quiz) => quiz.removeAttribute('daa-lh'));`
![image](https://github.com/adobecom/milo/assets/101133187/421b7d53-a3cc-43c0-a77f-059cfd23399d)

Reply:
According to the rule, we need to do this

Reply:
@vivgoodrich I've made these changes - have another look, please.